components/work_templates/index.tsx

```diff
@@ -88,6 +89,10 @@ const WorkTemplateModal = () => {
     const teamId = useSelector(getCurrentTeamId);
     const playbookTemplates = useSelector((state: GlobalState) => state.entities.worktemplates.playbookTemplates);
 
+    useEffect(() => {
+        trackEvent('work_templates', 'open_modal');
```

Comment:
is the value `open_modal` significant enough to keep track of the intention, I mean, is not too generic?

Reply:
I added it as a way to measure feature usage, because the modal opening = someone uses the feature.